### PR TITLE
Xcode 12.5 new notifications scan

### PR DESCRIPTION
**Pull request by AlexeyBudynkov**
1) Fix of two scripts: collect-all-alerts.rb, update-alerts.rb. Often they failed with this error: `glob': Interrupted system call - /some_path/...lproj (Errno::EINTR). After adding of a delay (0.005sec) the error has gone. Also added a number of attempts in the changed method.

2) Pulled additional alerts for Xcode 12.5

3) Added AppTransparency framework new notifications

### CODE REVIEW

**Comment by AlexeyBudynkov** [comment]
integration tests pipeline without corrupted cucumber start
https://msmobilecenter.visualstudio.com/Mobile-Center/_build/results?buildId=1069978&view=results

File: tools/springboard-alerts/collect-all-alerts.rb [hunk 1]
```diff
@@ -18,10 +18,22 @@ def add_entries_to_collection(lang, file_path, dict, collection)
 
 def collect_localization_dictionary(lang_dir_path)
   dict = {}
-  Dir.glob("#{lang_dir_path}/*.strings") do |file_path|
-    pairs = read_strings(file_path)
-    dict.merge!(pairs)
+  
+  3.times do |counter|
```
Comment (igorFedotenkov) [inline tools/springboard-alerts/collect-all-alerts.rb]: You don't use counter variable inside loop, so it could be removed:
3.times do ... end

**Review by igorFedotenkov — APPROVED** [review]

**Review by AndreyMaslennikov — CHANGES_REQUESTED** [review]
The original feature requires handling new alert about "access to advertising identifier" or something like this, so in my understanding we need to add handling for this alerts in this PR

**Review by AndreyMaslennikov — APPROVED** [review]

**Comment by AlexeyBudynkov** [comment]
Added AppTransparency framework new notifications

**Pull request merged**